modules/core/src/command_line_parser.cpp

```diff
@@ -4,6 +4,20 @@
 namespace cv
 {
 
+namespace {
+const char* noneValue = "<none>";
+
+String cat_string(const String& str)
```

Comment:
I was under the impression the the anonymous namespace implies static. Or is it because of your internal coding style?